app/controllers/untappdAuth.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const bodyParser = require('body-parser');
+
+function untappdAuth(req, res) {
+
+  let CODE = req.query.code
+  let CLIENTID = 'A0D5D7F766E859E3EF145BD051A3A576D2EA97CF'
+  let CLIENTSECRET = 'EBF811599C0F5914F7F37349041336C86926AC40'
+  let REDIRECT_URL = 'https://untappdtest.herokuapp.com/untappd-authentication'
+
+  fetch('https://untappd.com/oauth/authorize/?client_id=' + CLIENTID + '&client_secret=' + CLIENTSECRET + '&response_type=code&redirect_url=' + REDIRECT_URL + '&code=' + CODE , {
```

Comment:
To use this you should require the `node-fetch` module to use fetch like:
```suggestion
const fetch = require("node-fetch");
// some code
  fetch('https://untappd.com/oauth/authorize/?client_id=' + CLIENTID + '&client_secret=' + CLIENTSECRET + '&response_type=code&redirect_url=' + REDIRECT_URL + '&code=' + CODE , {
```